lib/raidenclient/RaidenClient.ts

```diff
@@ -201,12 +203,35 @@ class RaidenClient extends SwapClient {
     // not implemented, raiden does not use invoices
   }
 
-  public getRoutes =  async (_amount: number, _destination: string) => {
-    // stub placeholder, query routes not currently implemented in raiden
-    // assume a fixed lock time of 100 Raiden's blocks
-    return [{
+  public getRoutes =  async (amount: number, destination: string, currency: string) => {
```

Comment:
Normally I'd advocate an integration test here, but since this is temporary logic... :satisfied: